### PR TITLE
Service id in header

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

Adding service id as part of the request header

### Description of the Change

Changes in build modules and workflow
Changes in handlers
Changes in core apis
Changes in swagger file

### CODE REVIEW

File: core/jazz_codeq/config/test-config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"SONAR_USER": "{sonar_user}",
+	"SONAR_USER": "sonar_user",
```
Comment (pspani) [inline core/jazz_codeq/config/test-config.json]: why was this removed?

File: core/jazz_codeq/index.js [hunk 2]
```diff
@@ -82,7 +89,7 @@ function handler(event, context, cb) {
 					return cb(JSON.stringify(errorHandler.throwInputValidationError(messages.SERVICE_INPUT_ERROR)));
 				}
 			}).catch(err => {
-				logger.error(err.errorMessage);
+				logger.error("SINI CATCH "+ JSON.stringify(err));
```
Comment (pspani) [inline core/jazz_codeq/index.js]: change back

File: core/jazz_metrics/index.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ function handler(event, context, cb) {
     exportable.genericValidation(event)
       .then(() => validateUtils.validateGeneralFields(eventBody))
       .then(() => exportable.getToken(config))
-      .then((authToken) => exportable.getAssetsDetails(config, eventBody, authToken))
+      .then((authToken) => exportable.getAssetsDetails(config, eventBody, authToken, event.headers['Jazz-Service-ID']))
```
Comment (pspani) [inline core/jazz_metrics/index.js]: relevant update to getAssetsDetails function ?

File: core/jazz_codeq/config/test-config.json (hunk 1 above)
Comment (SiniWilson) [reply core/jazz_codeq/config/test-config.json]: just for testing I had changed. Updated.

File: core/jazz_codeq/index.js (hunk 2 above)
Comment (SiniWilson) [reply core/jazz_codeq/index.js]: done

File: core/jazz_metrics/index.js (hunk 3 above)
Comment (SiniWilson) [reply core/jazz_metrics/index.js]: Done
